public/extensions/breakout.js

```diff
@@ -0,0 +1,116 @@
+// @ts-check
+
+import daily from "./core.js";
+import Socket from "./socketiostate.js";
+
+let call;
+let socket;
+
+// Initial state
+let state = {
+  initialRoomUrl: "",
+  breakoutStarted: false,
+  participants: [{ user_name: "", roomUrl: "" }],
+};
+
+daily.beforeCreateFrame((parentEl, properties) => {
+  if (!properties.customTrayButtons) {
+    properties.customTrayButtons = {};
+  }
+
+  properties.customTrayButtons = {
+    ...properties.customTrayButtons,
+    toggleBreakout: {
+      iconPath: "https://www.svgrepo.com/show/207394/flash.svg",
```

Comment:
I imagine Christian will want this to be a black-and-white SVG eventually, but we need to hassle him to give us light-mode and dark-mode options I guess :)